packages/components/src/Actions/ActionDropdown/ActionDropdown.component.md

```diff
@@ -1,176 +1,115 @@
-components: {
-	beforeItemsDropdown: [] or {},
-	itemsDropdown: [] or {},
-	afterItemsDropdown: [] or {},
-}
-``` 
-They are 3 'placeholders' in the component to inject new stuff.
-'itemsDropdown' is the classic content list of the dropdown, equivalent to where the items props will be rendered.
-And you have a 'beforeItemsDropdown' and 'afterItemsDropdown' key, that will be rendered before or after the itemsDropdown in the list, if you need specific stuff that will give you more flexibility.
 
+Using this you can defined additionnal component that your `ActionDropdown` will show.
+(This do no permit to alter the render of `items`)
 
-The object and the items of the array must have at least a component attribute ```{ component: 'MyComponentName' }```.
+They are 3 slots in the `ActionDropdown` component where you can inject your custom components.
 
-You can create an object with a divider and no component. It will render a line between items.
-```javascript
-<MenuItem divider />
-```
+* `beforeItemsDropdown` will allow you to inject one or many components before the item list
+* `itemsDropdown` will allow you to inject one or many components just after the item list
+* `afterItemsDropdown` will allow you to inject one or many components before the item list (this should be deprecated since this exactly what `itemsDropdown` will do)
 
-The components in ActionDropdown supports also a special prop 'withMenuItem' which is a boolean.
-If it's true your component will be wrapped
-```javascript
-<MenuItem><YourComponent /></MenuItem>
-```
-The MenuItem component came from react-bootstrap.
-If false or undefined you will get
-```javascript
-<li><YourComponent /></li>
-```
+Each of this component should be defined as an object using the following api :
+
+| properties    | type                                                                                       |
+| ------------- | ------------------------------------------------------------------------------------------ |
+| component     | String (Required if not a divider)                                                         |
+| divider       | Bool                                                                                       |
+| withMenuItem  | Bool                                                                                       |
+| menuItemProps | [react-bootstrap](https://react-bootstrap.github.io/components/dropdowns/#menu-item-props) |
+| liProps       | Object                                                                                     |
 
-The ```<li>``` and ```<MenuItem>``` can be deeply customized with menuItemProps and liProps.
+Your component will be wrapped into a `MenuItem` if `withMenuItem` property is set to true
+This will give you access to customisation of [react-bootstrap](https://react-bootstrap.github.io/components/dropdowns/#menu-items) `MenuItem` component
+
+The `<li>` and `<MenuItem>` can be deeply customized with menuItemProps and liProps.
```

Comment:
With the LI we are loosing some stuff from react bootstrap like the accessibility with the keyboard shortcut for example.
If we decide to keep it like that, you can maybe precise it in the documentation.